JenkinsfileDeploy

```diff
@@ -0,0 +1,97 @@
+node('docker') {
+
+    // Global variables
+    String DOCKER_IMAGE_VERSION = '8491d157'
```

Comment:
Wdyt about adding DOCKER_IMAGE_VERSION to input?

Reply:
if it's possible to let it idle ? In the way that no input => latest by default or other version

Reply:
imo, if not input, abort
(?)

Reply:
okay !